routers/web/repo/http.go

```diff
@@ -494,6 +494,8 @@ func serviceRPC(h serviceHandler, service string) {
 	cmd := exec.CommandContext(ctx, git.GitExecutable, service, "--stateless-rpc", h.dir)
 	cmd.Dir = h.dir
 	cmd.Env = append(os.Environ(), h.environ...)
+	gArgs := git.GlobalCommandArgs
```

Comment:
https://go.dev/play/p/n07qz89-2Jb vs https://go.dev/play/p/xF4eSYAKqIZ

:thinking: - changing ...

Reply:
for basic types `:=` works https://go.dev/play/p/j5kcNfR6bpe